<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundEffectDialogClass</class>
 <widget class="QDialog" name="SoundEffectDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>189</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <weight>50</weight>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>音效</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="eqBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>均衡器</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="lowLabel">
      <property name="text">
       <string>125Hz</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSlider" name="midFreqSlider">
      <property name="styleSheet">
       <string notr="true">QSlider::groove:vertical {                                
     border: 1px solid #999999;                  
     height: 105px;                                           
     margin: 1px;                                         
     left: 8px; right:8px;
     bottom: 30px; top:30 px;
 }                                                          
QSlider::handle:vertical {                                                
     border: 1px solid #5c5c5c;                
     border-image:url(../Resources/slider.png);
     width:20px;                                             
     margin: -10px -10px;                           
 }
QSlider::add-page:vertical {
        background-color: rgb(37, 168, 198);
        width:4px;
}
QSlider::sub-page:vertical {
        background-color: rgb(87, 97, 106);
        width:4px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QSlider" name="reverbSlider">
      <property name="styleSheet">
       <string notr="true">QSlider::groove:vertical {                                
     border: 1px solid #999999;                  
     height: 105px;                                           
     margin: 1px;                                         
     left: 8px; right:8px;
     bottom: 25px; top:25 px;
 }                                                          
QSlider::handle:vertical {                                                
     border: 1px solid #5c5c5c;                
     border-image:url(../Resources/slider.png);
     width:10px;                                             
     margin:-10px -10px;                           
 }
QSlider::add-page:vertical {
        background-color: rgb(37, 168, 198);
        width:4px;
}
QSlider::sub-page:vertical {
        background-color: rgb(87, 97, 106);
        width:4px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QSlider" name="highFreqSlider">
      <property name="styleSheet">
       <string notr="true">QSlider::groove:vertical {                                
     border: 1px solid #999999;                  
     height: 105px;                                           
     margin: 1px;                                         
     left: 8px; right:8px;
     bottom: 30px; top:30 px;
 }                                                          
QSlider::handle:vertical {                                                
     border: 1px solid #5c5c5c;                
     border-image:url(../Resources/slider.png);
     width:20px;                                             
     margin: -10px -10px;                           
 }
QSlider::add-page:vertical {
        background-color: rgb(37, 168, 198);
        width:4px;
}
QSlider::sub-page:vertical {
        background-color: rgb(87, 97, 106);
        width:4px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="midLabel">
      <property name="text">
       <string>1KHz</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLabel" name="reverbLabel">
      <property name="text">
       <string>混响</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="highLabel">
      <property name="text">
       <string>8KHz</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QSlider" name="lowFreqSlider">
      <property name="styleSheet">
       <string notr="true">QSlider::groove:vertical {                                
     border: 1px solid #999999;                  
     height: 105px;                                           
     margin: 1px;                                         
     left: 8px; right:8px;
     bottom: 30px; top:30 px;
 }                                                          
QSlider::handle:vertical {                                                
     border: 1px solid #5c5c5c;                
     border-image:url(../Resources/slider.png);
     width:20px;                                             
     margin: -10px -10px;                           
 }
QSlider::add-page:vertical {
        background-color: rgb(37, 168, 198);
        width:4px;
}
QSlider::sub-page:vertical {
        background-color: rgb(87, 97, 106);
        width:4px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSlider" name="midhighFreqSlider">
      <property name="styleSheet">
       <string notr="true">QSlider::groove:vertical {                                
     border: 1px solid #999999;                  
     height: 105px;                                           
     margin: 1px;                                         
     left: 8px; right:8px;
     bottom: 30px; top:30 px;
 }                                                          
QSlider::handle:vertical {                                                
     border: 1px solid #5c5c5c;                
     border-image:url(../Resources/slider.png);
     width:20px;                                             
     margin: -10px -10px;                           
 }
QSlider::add-page:vertical {
        background-color: rgb(37, 168, 198);
        width:4px;
}
QSlider::sub-page:vertical {
        background-color: rgb(87, 97, 106);
        width:4px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="lowmidFreqSlider">
      <property name="styleSheet">
       <string notr="true">QSlider::groove:vertical {                                
     border: 1px solid #999999;                  
     height: 105px;                                           
     margin: 1px;                                         
     left: 8px; right:8px;
     bottom: 30px; top:30 px;
 }                                                          
QSlider::handle:vertical {                                                
     border: 1px solid #5c5c5c;                
     border-image:url(../Resources/slider.png);
     width:20px;                                             
     margin: -10px -10px;                           
 }
QSlider::add-page:vertical {
        background-color: rgb(37, 168, 198);
        width:4px;
}
QSlider::sub-page:vertical {
        background-color: rgb(87, 97, 106);
        width:4px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QSlider" name="extremhighFreqSlider">
      <property name="styleSheet">
       <string notr="true">QSlider::groove:vertical {                                
     border: 1px solid #999999;                  
     height: 105px;                                           
     margin: 1px;                                         
     left: 8px; right:8px;
     bottom: 30px; top:30 px;
 }                                                          
QSlider::handle:vertical {                                                
     border: 1px solid #5c5c5c;                
     border-image:url(../Resources/slider.png);
     width:20px;                                             
     margin: -10px -10px;                           
 }
QSlider::add-page:vertical {
        background-color: rgb(37, 168, 198);
        width:4px;
}
QSlider::sub-page:vertical {
        background-color: rgb(87, 97, 106);
        width:4px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="lowmidLabel">
      <property name="text">
       <string>400Hz</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="midhighLabel">
      <property name="text">
       <string>3KHz</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="extremhighLabel">
      <property name="text">
       <string>12KHz</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="sqGroupBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>DSP</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QRadioButton" name="rotateRadioButton">
      <property name="text">
       <string>旋转</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="echoRadioButton">
      <property name="text">
       <string>回声</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="flangerRadioButton">
      <property name="text">
       <string>飘忽</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="confirmPushBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
